### PR TITLE
Managing that progressbar

### PR DESCRIPTION
**Pull request by heysadboy**
Fix for the issue https://github.com/fossasia/open-event-android/issues/1087 . Please see the issue to know more.

### CODE REVIEW

File: android/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -36,18 +36,17 @@
...
-            android:id="@+id/progress"
-            style="@style/Base.Widget.AppCompat.ProgressBar.Horizontal"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="?attr/actionBarSize"
-            android:max="100"
-            android:visibility="gone" />
+            <ProgressBar
+                android:id="@+id/progress"
+                android:layout_marginBottom="-6dp"
```
Comment (the-dagger) [inline android/app/src/main/res/layout/activity_main.xml]: Please move this to dimens instead, rather than hardcoding the values :slightly_smiling_face:

Comment (heysadboy) [inline android/app/src/main/res/layout/activity_main.xml]: okay, thanks for looking up

**Comment by heysadboy** [comment]
@the-dagger Here we go, added values to dimens

**Comment by the-dagger** [comment]
Just wanted to clarify, as to why this is done only in the track list though?
We have other places in the app too where we face this issue.

**Comment by heysadboy** [comment]
@the-dagger Sorry, actually the other activity in which I found progress bar is activity_speakers, other than track list and that is a circular one. It'd be great if you could tell if you notice it anywhere else.

**Comment by the-dagger** [comment]
It's there in the sponsors,locations and the maps fragment too as far as I
know.

**Comment by heysadboy** [comment]
@the-dagger Actually the progress bar is in activity_main only, fragments don't have it separately. So it will be like that everywhere, it's kinda shared between them. I checked the usages of ProgressBar and I only found it only in activity_main and activity_speakers.

**Comment by the-dagger** [comment]
Oh yeah, you are right.
Sorry, my bad!
In this case, this PR is alright then :+1:

**Pull request merged**